ifrs17/Report/ReportMutableScopes.ipynb

```diff
@@ -115,7 +115,9 @@
                 "\n    string Scenario { get; set; }  // TODO: Enable dropdown selection including All and Delta",
                 "\n    string Comparison { get; set; }  // TODO: only for scenario at the beginning, meant to enable general purpose comparisons ",
                 "\n    CurrencyType CurrencyType { get; set; }",
-                "\n    ",
+                "\n    int Projection {get; set;}",
+                "\n    protected ProjectionConfiguration[] projectionConfigurations => GetStorage().ProjectionConfigurations;",
```

Comment:
I think you can apply the FilterAndOrder(Month) here as this is a mutable scope and teh Month is a property which can be set from the outside.